src/ophys_etl/modules/suite2p_wrapper/schemas.py

```diff
@@ -86,6 +86,53 @@ class Suite2PWrapperSchema(argschema.ArgSchema):
                     "of computing one internally. To use automated reference "
                     "image generation in ophys_etl, set this value to True and"
                     "refImg to an empty list or array (Default).")
+    do_optimize_motion_params = argschema.fields.Bool(
+        default=False,
+        required=False,
+        description="Do a search for best parameters of smooth_sigma and "
+                    "smooth_sigma_time. Adds significant runtime cost to "
+                    "motion correction and should only be run once per "
+                    "experiment with the resulting parameters being stored "
+                    "for later use.")
+    use_ave_image_as_reference = argschema.fields.Bool(
+        default=False,
+        required=False,
+        description="Only available if `do_optimize_motion_params` is set. "
+                    "After the a best set of smoothing parameters is found, "
+                    "use the resulting average image as the reference for the "
+                    "full registration. This can be used as two step "
+                    "registration by setting min=max in both parameters and "
```

Comment:
I don't understand what you mean by "by setting min=max in both parameters"

Reply:
I see...
Maybe "by setting smooth_sigma_min=smooth_sigma_max and smooth_sigma_time_min=smooth_sigma_time_max", just to avoid all possible confusion?